proto/eraftpb.proto

```diff
@@ -91,11 +94,20 @@ enum ConfChangeType {
     AddNode    = 0;
     RemoveNode = 1;
     AddLearnerNode = 2;
+    BeginMembershipChange = 3;
+    FinalizeMembershipChange = 4;
 }
 
 message ConfChange {
     uint64 id = 1;
     ConfChangeType change_type = 2;
+    // Used in `AddNode`, `RemoveNode`, and `AddLearnerNode`.
     uint64 node_id = 3;
     bytes context = 4;
+    // Used in `BeginMembershipChange` and `FinalizeMembershipChange`.
+    ConfState configuration = 5;
```

Comment:
If you agree my comment about `ConfState`. Here I think we should add a new struct, for example:
```protobuf
struct MembershipChange {
    repeated uint64 nodes;
    repeated uint64 learner_nodes;
}
```

Reply:
😆 This is exactly what `ConfState` was originally.